digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/csvDataImporter/PedestrianCrossingCsvImporter.scala

```diff
@@ -27,8 +27,7 @@ class PedestrianCrossingCsvImporter(roadLinkServiceImpl: RoadLinkService, eventB
 
       val position = getCoordinatesFromProperties(csvProperties)
 
-      val roadLink = roadLinkService.enrichRoadLinksFromVVH(nearbyLinks)
```

Comment:
Dont we need the enrichedLinks here?

Reply:
getClosestRoadlinkForCarTrafficFromVVH now enriches the VVH roadlinks with oth information so the nearbyLinks do not need to enrich it again